static/docs/use-cases/data-registries.md

```diff
@@ -0,0 +1,195 @@
+# Data Registries
```

Comment:
I tend to agree with @Suor - still not sure why do we use Registries here instead of Registry

Reply:
This is something we decided privately [in Slack](https://iterativeai.slack.com/archives/DLW4G0F7W/p1576028094019200?thread_ts=1576025894.008300&cid=DLW4G0F7W). Although your idea originally was **Data and model registries**, but we ended only making it plural for data, and adding more mentions of models in the body. 

BTW I have changed the nav title to plural also already: https://dvc-landing-use-cases-d-b6xlg5.herokuapp.com/doc/use-cases/data-registries. Should we roll back both to singular?

Reply:
@jorgeorpinel I will let you decide on this :) to me "data registry" sounds better than "data registries" as I'm reading it now ... model - you decided (and that was probably right) to not complicate it

Reply:
I actually like the plural better. I think otherwise it may sound like we only recommend concentrating everything in a single registry, whereas one could have several ones, some for datasets, some for models, etc.